src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -182,49 +181,55 @@ export const PropertyList = ({
         dispatch({ type: SET_OAT_PROPERTY_EDITOR_MODEL, payload: newModel });
     };
 
+    const handleSelectorPosition = (e) => {
+        const boundingRect = e.target.getBoundingClientRect();
+        setPropertySelectorPosition({
+            ...propertySelectorPosition,
+            top: boundingRect.top,
+            left: boundingRect.left
+        });
+        setPropertySelectorTriggerElementsBoundingBox(boundingRect);
+    };
+
+    const getIsMouseLeaveNotNorth = (e) => {
+        if (
+            e.clientY >= propertySelectorTriggerElementsBoundingBox.bottom ||
+            e.clientX < propertySelectorTriggerElementsBoundingBox.left ||
+            e.clientX > propertySelectorTriggerElementsBoundingBox.right
+        ) {
+            return true;
+        }
+
+        return false;
+    };
```

Comment:
Given this function is passed as a prop to another function, we should probably wrap it in a `useCallback` so it only changes when the state property changes.

Or, perhaps a cleaner solution would be to pass the state (i.e. the bounding box) as a prop and define this function outside of the component and export it?